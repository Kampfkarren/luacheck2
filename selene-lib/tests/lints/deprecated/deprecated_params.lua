local _ = Instance.new(a)
local _ = Instance.new(a, b)
local _ = Instance.new(a,    nil     )
local _ = Instance.new(a, "nil")

a(1)
a ""
a {}
