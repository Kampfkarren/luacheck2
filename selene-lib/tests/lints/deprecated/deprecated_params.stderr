error[deprecated]: this parameter is deprecated
  ┌─ deprecated_params.lua:2:27
  │
2 │ local _ = Instance.new(a, b)
  │                           ^
  │
  = set the instance's parent separately

error[deprecated]: this parameter is deprecated
  ┌─ deprecated_params.lua:4:27
  │
4 │ local _ = Instance.new(a, "nil")
  │                           ^^^^^
  │
  = set the instance's parent separately

error[deprecated]: this parameter is deprecated
  ┌─ deprecated_params.lua:6:3
  │
6 │ a(1)
  │   ^
  │
  = this is deprecated

error[deprecated]: this parameter is deprecated
  ┌─ deprecated_params.lua:7:3
  │
7 │ a ""
  │   ^^
  │
  = this is deprecated

error[deprecated]: this parameter is deprecated
  ┌─ deprecated_params.lua:8:3
  │
8 │ a {}
  │   ^^
  │
  = this is deprecated

